spec/rails_helper.rb

```diff
@@ -1,6 +1,5 @@
 ENV['RAILS_ENV'] ||= 'test'
 
-require 'spec_helper'
```

Comment:
Why are you removing the `spec_helper` here?

Reply:
Isn't needed

Reply:
So we return open struct in the api which is convenient but difficult to stub in the tests since we weren't allowing unverifying doubles. I'd literally have to define each level of OpenStruct as an object and fill it with what I needed. My choices were:

1. Move to pure json - lose the convenience of open struct
2. Remove the verifying partial line in the spec - cleaner but we're not even using the spec helper atm
3. Add all of the open struct definitions - painful and verbose and not adding a lot of value
4. Remove the spec helper